docs/source/releases/latest/608-documentation-system-requirements-for-geoips.yaml

```diff
@@ -0,0 +1,15 @@
+documentation:
+- description: 'Added system requirements to the documentation. This includes minimum and recommended requirements for users and developers as well as language limiting the applicability of the system requirements.'
```

Comment:
Should these typically be written as

```
- description: |
    Added system requirements ...
    This includes ...
```

Rather than the long descriptions on a single line? Then the RST files will be within the recommended line length limit?